docs/levenscyclus/index.md

```diff
@@ -1,32 +1,37 @@
 --- 
 title: Levenscyclus
+summary: Hulp per fase van de levenscyclus van algoritmes en AI, zoals probleemanalyse, ontwerpen, ontwikkelen, implementeren en ermee stoppen.
 icon: material/reload
 --- 
+# Levenscyclus algoritmes en AI
 
-!!! info "Disclaimer"
+Verantwoordelijke algoritmes en AI voldoen tijdens elke fase van de levenscyclus aan de regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
 
-    Het Algoritmekader is nog volop in ontwikkeling. Op deze plek willen we vooral aan de slag gaan op een open en transparante wijze. Het is dus niet definitief. Dat betekent dat er dingen opstaan die niet af zijn en soms zelfs fout. Mocht er iets niet kloppen, laat het ons weten via [GitHub](https://github.com/MinBZK/Algoritmekader).
+## Fases van de levenscyclus
```

Comment:
Deze staan ook al in de figuur en links in het menu?

Reply:
Figuur en linker-menu zou ik graag verwijderen.